crates/uv/src/main.rs

```diff
@@ -445,8 +445,14 @@ struct PipSyncArgs {
     /// any parent directory. The `--python` option allows you to specify a different interpreter,
     /// which is intended for use in continuous integration (CI) environments or other automated
     /// workflows.
-    #[clap(long)]
-    python: Option<PathBuf>,
+    ///
+    /// Supported formats:
+    /// - `3.10` looks for an installed Python 3.10 using `py --list-paths` on Windows, or
+    ///   `python3.10` on Linux and macOS. (Specifying a patch version is not supported.)
```

Comment:
Patch versions are supported, `uv pip install --python 3.11.6` passes, `uv pip install --python 3.11.5` errors.

Reply:
Ah ok. This was copied over from the other messages in the CLI. It must be stale. I'll fix it separately.